client/app/components/SkipToContent/index.jsx

```diff
@@ -0,0 +1,24 @@
+// @flow
+import React from 'react';
+
+import css from './SkipToContent.scss';
+import { I18n } from '../../libs/i18n';
+
+export type Props = {
+  id: string,
+};
+
+function SkipToContent(props: Props) {
+  const { id } = props;
+
+  window.addEventListener('Props', () => {
+    window.scrollTo(0, 0)});
```

Comment:
I think we're missing a `}` here.

I'm not sure what listening to "Props" actually does. I would assume it's listening to all prop changes? I don't think we want to do that.

Instead, I think we want to put an `onClick` listener on the link element below and put the scroll logic there.

Reply:
Hey @julianguyen ! Thanks for the feedback. We started working on that this week.

Would you like us to refactor our current SkipToContent function into a class component (in order to add the scroll logic)? Or could we also use react hooks? 

I'm asking because we're not sure if react hooks can be used in a react on rails context. 

Thank you!

Reply:
You can use React Hooks here! But if you feel more comfortable using React classes, that's fine too.